ethereum/jsonrpc/src/main/java/tech/pegasys/pantheon/ethereum/jsonrpc/internal/results/TransactionReceiptResult.java

```diff
@@ -81,7 +81,11 @@ public TransactionReceiptResult(final TransactionReceiptWithMetadata receiptWith
     this.to = receiptWithMetadata.getTransaction().getTo().map(BytesValue::toString).orElse(null);
     this.transactionHash = receiptWithMetadata.getTransaction().hash().toString();
     this.transactionIndex = Quantity.create(receiptWithMetadata.getTransactionIndex());
-    this.revertReason = receipt.getRevertReason().orElse(null);
+    this.revertReason =
+        receipt
+            .getRevertReason()
+            .map(reason -> reason.isEmpty() ? "" : reason.toString())
```

Comment:
I think this should just be:
```suggestion
            .map(BytesValue::toUnprefixedString)
```
as we want to output it as a hex value in the same way as `StructLog` does above.

Reply:
that means you get "revertReason": "0x" ... which ... seemed odd to me - but if thats the preferred solution ...